src/aap_eda/settings/post_load.py

```diff
@@ -0,0 +1,412 @@
+        },
+        "root": {"handlers": ["console"], "level": "WARNING"},
+        "loggers": {
+            "django": {
+                "handlers": ["console"],
+                "level": "WARNING",
+                "propagate": False,
+            },
+            "django.request": {
+                "handlers": ["console"],
+                "level": settings.APP_LOG_LEVEL,
+                "propagate": False,
+            },
+            "django.channels.server": {
+                "handlers": ["console"],
+                "level": settings.APP_LOG_LEVEL,
+                "propagate": False,
+            },
+            "aap_eda": {
+                "handlers": ["console"],
+                "level": settings.APP_LOG_LEVEL,
+                "propagate": False,
+            },
+            "ansible_base": {
+                "handlers": ["console"],
+                "level": settings.APP_LOG_LEVEL,
+                "propagate": False,
+            },
+        },
+    }
+
+
+def _get_default_pg_notify_dsn_server(settings: Dynaconf) -> str:
+    db_options = settings.DATABASES["default"].get("OPTIONS", {})
+    return (
+        f"host={settings.DATABASES['default']['HOST']} "
+        f"port={settings.DATABASES['default']['PORT']} "
+        f"dbname={settings.DATABASES['default']['NAME']} "
+        f"user={settings.DATABASES['default']['USER']} "
+        f"password={settings.DATABASES['default']['PASSWORD']} "
+        f"sslmode={db_options.get('sslmode','allow')} "
+        f"sslcert={db_options.get('sslcert','')} "
+        f"sslkey={db_options.get('sslkey','')} "
+        f"sslrootcert={db_options.get('sslrootcert','')} "
+    )
+
+
+def _set_resource_server(settings: Dynaconf) -> None:
+    if not settings.RESOURCE_SERVER:
+        settings.RESOURCE_SERVER = {
+            "URL": settings.RESOURCE_SERVER__URL,
+            "SECRET_KEY": settings.RESOURCE_SERVER__SECRET_KEY,
+            "VALIDATE_HTTPS": settings.RESOURCE_SERVER__VALIDATE_HTTPS,
+        }
+
+    if (
+        settings.RESOURCE_SERVER["URL"]
+        and settings.RESOURCE_SERVER["SECRET_KEY"]
+    ):
+        settings.RQ_PERIODIC_JOBS.append(
+            {
+                "func": "aap_eda.tasks.shared_resources.resync_shared_resources",  # noqa: E501
+                "interval": 900,
+                "id": "resync_shared_resources",
+            }
+        )
+
+
+def post_loading(settings: Dynaconf):
+    settings.SECRET_KEY = _get_secret_key(settings)
+    settings.DEBUG = get_boolean(settings, "DEBUG")
+    settings.ALLOWED_HOSTS = _get_list_from_str(settings, "ALLOWED_HOSTS")
+    # A list or a comma separated string of allowed origins for CSRF protection
+    # in the form of [scheme://]host[:port]. Supports wildcards.
+    # More info: https://docs.djangoproject.com/en/4.2/ref/settings/#csrf-trusted-origins  # noqa: E501
+    settings.CSRF_TRUSTED_ORIGINS = _get_list_from_str(
+        settings, "CSRF_TRUSTED_ORIGINS"
+    )
+    toggle_feature_flags(settings)
```

Comment:
Toggle feature flags is called later at the end of this function, is calling it twice needed?

Reply:
good catch. Will remove this line